docs/design/elastic.md

```diff
@@ -0,0 +1,100 @@
+# 弹性训练设计
```

Comment:
By default a english version doc is needed. Besides a chinese version would be even better.